src/app/random-name/names.ts

```diff
@@ -0,0 +1,8 @@
+export interface Name {
```

Comment:
Czy te interfejsy są poprawne? Z tego co widzę to 
Name: string;
NamesData: string[];

Chyba ze to gdzieś przemapowujesz.

Reply:
Zostawiłem tylko interfejs Name { value: string }

Reply:
Napisałem Ci nowy komentarz. Name nie ma pola value. Name jest stringiem.

export type Name = string;

Reply:
Interfejs przepisany: 
export interface Name = string

export interface Names = Name[]

Reply:
Czyli tu nie "interface" a "type"... poprawione